matching-notification-api/src/main/java/org/phenotips/matchingnotification/match/internal/DefaultPhenotypesMap.java

```diff
@@ -219,17 +219,29 @@ public static void reorder(List<Map<String, String>> predefined1, List<Map<Strin
             if (StringUtils.equals(name1, name2)) {
                 common1.add(item1);
```

Comment:
@allasm @sdumitriu  I made an alternative implementation of this reordering function in the next commit to avoid exclusions and make process more simple & clear.